gh2/csv.py

```diff
@@ -207,24 +209,26 @@ def set_headers(repo, labels=None):
     return headers
 
 
-def main():
-    parser = make_parser()
+def get_all_labels(repositories):
```

Comment:
Ah, I'd missed the issue_to_dict bit but now I look at it, this patch introduces a bug. If you look at line 90 it tests for the label using the label object but that wouldn't work if a label could be from either repo.